.github/workflows/build.yaml

```
@@ -182,6 +191,15 @@ jobs:
   build_vm:
     runs-on: ubuntu-24.04
     container: espressif/idf:v5.4
+    strategy:
+      matrix:
+        size: ["16MB", "4MB"]
+      include:
```

actionlint (1.7.10)

[error] unexpected key "include" for "strategy" section. expected one of "fail-fast", "matrix", "max-parallel"

(syntax-check)